deploy/csv-templates/crds/ocs.openshift.io_storageclusters_crd.yaml

```diff
@@ -1223,6 +1223,10 @@ spec:
                       when the actual configurable options are defined in rook-ceph'
                     type: object
                     properties:
+                      tuneFastDeviceClass:
```

Comment:
This got added after `make gen-latest-csv`

Reply:
That is weird.... oh, I bet the CSV test is passing because it wasn't fixed for this branch. Okay, I guess.